CONTRIBUTING.md

```diff
@@ -115,6 +115,22 @@ For these test executables, `-p` which applies a regex filter to the test
 names. When running `cabal-install` test suites, one need only use `cabal test` or
 `cabal run <test-target>` in order to test locally.
 
+QA Notes
+--------
+
+Manual Quality Assurance (QA) is performed to ensure that the changes impacting the command-line interface, whether adding or modifying a behaviour,
```

Comment:
I think line-breaking approach in the added text diverges from what is used in the file (at first glance -- I didn't apply a formatter...). Can you reflow so that the lines are approximately the same length as all else?

Reply:
Yep!